src/platform/tigerlake/include/platform/lib/pm_runtime.h

```diff
@@ -18,12 +18,7 @@
 
 #include <stdint.h>
 
-struct pm_runtime_data;
-
-/** \brief Platform specific runtime power management data. */
-struct platform_pm_runtime_data {
-	/* TBD */
-};
+#include <cavs/lib/pm_runtime.h>
```

Comment:
Does it make sense for cAVS platforms to also move function declarations to cavs/lib/pm_runtime.h?
Less files to change in case of adding new function e.g. `platform_pm_runtime_active`.

Reply:
In general this is a good idea, however this function has a different declaration for other platforms, so kept it in the individual platform headers for symmetry.